infrastructure/template.yaml

```
@@ -1537,8 +1552,12 @@ Resources:
           - [!Ref LogRedactionFunctionCanaryErrorAlarm, !Ref LogRedactionFunctionFatalErrorAlarm]
           - [!Ref AWS::NoValue]
         Role: !GetAtt CodeDeployServiceRole.Arn
-
+      Environment:
+        Variables:
+          LogStreamTrackingTbl: !Ref LogStreamTrackingTbl
```

```suggestion
          LogStreamTrackingTable: !Ref LogStreamTrackingTable
```
Please use full name for "table" in line with other templates